src-view/components/ProfileCard.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import cx from "clsx";
+import Box from "@material-ui/core/Box";
+import Card from "@material-ui/core/Card";
+import Button from "@material-ui/core/Button";
+import CardContent from "@material-ui/core/CardContent";
+import Avatar from "@material-ui/core/Avatar";
+import Divider from "@material-ui/core/Divider";
+import { useFadedShadowStyles } from "@mui-treasury/styles/shadow/faded";
+import { useGutterBorderedGridStyles } from "@mui-treasury/styles/grid/gutterBordered";
+import { grey, purple, teal } from "@material-ui/core/colors";
+import { createMuiTheme, makeStyles } from "@material-ui/core/styles";
+
+const theme = createMuiTheme({
+  palette: {
+    primary: grey,
+  },
+});
+const useStyles = makeStyles(({ palette }) => ({
```

Comment:
I would suggest the use of modular scss. Like the ones done in other components.

Reply:
Also use flex or grid for responsiveness. Want the keep the dependency on semantic UI as less as possible. That is why we have started with setting up a design system.